2021-05-27 06:06:48,806 INFO    MainThread:16497 [wandb_setup.py:_flush():69] setting env: {}
2021-05-27 06:06:48,806 INFO    MainThread:16497 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-27 06:06:48,806 INFO    MainThread:16497 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210527_060648-3a6vfa6o/logs/debug.log
2021-05-27 06:06:48,806 INFO    MainThread:16497 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210527_060648-3a6vfa6o/logs/debug-internal.log
2021-05-27 06:06:48,806 INFO    MainThread:16497 [wandb_init.py:init():369] calling init triggers
2021-05-27 06:06:48,806 INFO    MainThread:16497 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-05-27 06:06:48,806 INFO    MainThread:16497 [wandb_init.py:init():418] starting backend
2021-05-27 06:06:48,806 INFO    MainThread:16497 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-27 06:06:48,809 INFO    MainThread:16497 [backend.py:ensure_launched():132] starting backend process...
2021-05-27 06:06:48,840 INFO    MainThread:16497 [backend.py:ensure_launched():136] started backend process with pid: 17002
2021-05-27 06:06:48,842 INFO    MainThread:16497 [wandb_init.py:init():423] backend started and connected
2021-05-27 06:06:48,844 INFO    MainThread:16497 [wandb_init.py:init():465] updated telemetry
2021-05-27 06:06:48,844 INFO    MainThread:16497 [wandb_init.py:init():484] communicating current version
2021-05-27 06:06:49,623 INFO    MainThread:16497 [wandb_init.py:init():489] got version response 
2021-05-27 06:06:49,624 INFO    MainThread:16497 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-27 06:06:49,841 INFO    MainThread:16497 [wandb_init.py:init():522] starting run threads in backend
2021-05-27 06:06:49,877 INFO    MainThread:16497 [wandb_run.py:_console_start():1565] atexit reg
2021-05-27 06:06:49,878 INFO    MainThread:16497 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-27 06:06:49,878 INFO    MainThread:16497 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-27 06:06:49,880 INFO    MainThread:16497 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-27 06:06:49,880 INFO    MainThread:16497 [wandb_init.py:init():546] run started, returning control to user process
2021-05-27 06:06:49,884 INFO    MainThread:16497 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 0
2021-05-27 06:06:49,885 INFO    MainThread:16497 [wandb_run.py:_restore():1507] restore
2021-05-27 06:06:51,890 INFO    MainThread:16497 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 719
  total_bytes: 719
}

2021-05-27 06:06:52,395 INFO    MainThread:16497 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 719
  total_bytes: 719
}

2021-05-27 06:06:52,846 INFO    MainThread:16497 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 719
  total_bytes: 3670
}

2021-05-27 06:06:53,500 INFO    MainThread:16497 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3780
  total_bytes: 3780
}

2021-05-27 06:06:53,603 INFO    MainThread:16497 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3780
  total_bytes: 3780
}

2021-05-27 06:06:53,705 INFO    MainThread:16497 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3780
  total_bytes: 3780
}

2021-05-27 06:06:53,807 INFO    MainThread:16497 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3780
  total_bytes: 3780
}

2021-05-27 06:06:53,908 INFO    MainThread:16497 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3780
  total_bytes: 3780
}

2021-05-27 06:06:54,992 INFO    MainThread:16497 [wandb_run.py:_show_summary():1812] rendering summary
2021-05-27 06:06:54,992 INFO    MainThread:16497 [wandb_run.py:_show_history():1850] rendering history
2021-05-27 06:06:54,992 INFO    MainThread:16497 [wandb_run.py:_show_files():1879] logging synced files
